.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+jdk: oraclejdk8
+
+script:
+  - cd alma-server/
```

Comment:
file should be modified to have the following content : 
script:
  - mvn clean verify -f alma-server/pom.xml -e